src/index.js

```diff
@@ -364,6 +366,32 @@ const initialConnectedButtons = [
   maliciousSeaport,
 ];
 
+// Buttons that are available after connecting via Wallet Connect
```

Comment:
not the same as the ones with Extension, as we don't allow to deploy contracts for now (more logic might be needed for getting deployed contract address)